src/Linux/R/Interpreters/Test/RInstallationTest.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.Common.Core;
+using Microsoft.UnitTests.Core.Linux;
+using Xunit;
+
+namespace Microsoft.R.Interpreters.Linux.Test {
+    public class RInstallationTest {
+        [Fact]
+        public void RInstallationBasicTest() {
+            var fs = new TestFileSystem();
+            var rInstallation = new RInstallation(fs);
+            var installs = rInstallation.GetCompatibleEngines();
```

Comment:
The current test file results in this order, if we update the test file in the future the order may change. In the Linux package status file, the order is not determinate.